### PR TITLE
Add UT for testing EnableProfiling in scheduler

### PR DESCRIPTION
**Pull request by mysunshine92**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add UT for testing EnableProfiling in scheduler

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kube-scheduler/app/server_test.go [hunk 1]
```diff
@@ -287,3 +295,44 @@ profiles:
...
+
+func TestNewHealthzHandler(t *testing.T) {
+	// Setup any healthz checks we will want to use.
+	var checks []healthz.HealthChecker
+
+	//Test1: EnableProfiling=true
+	c := &kubeschedulerconfig.KubeSchedulerConfiguration{
+		DebuggingConfiguration: componentbaseconfig.DebuggingConfiguration{
+			EnableProfiling:           true,
+			EnableContentionProfiling: true,
```
Comment (chendave) [inline cmd/kube-scheduler/app/server_test.go:307]: Looks like you mostly want to test the enabling of profiling? if that is the case the name of the testcase and pr subject should be updated accordingly.

Comment (mysunshine92) [reply cmd/kube-scheduler/app/server_test.go:307]: thanks，i have updated.

Comment (alculquicondor) [reply cmd/kube-scheduler/app/server_test.go:307]: Make it a testtable

**Comment by dims** [comment]
Is this PR still needed, please rebase if so (or we can close it?)

**Comment by dims** [comment]
This PR is older than 4 weeks and needs as rebase.

Please rebase the PR against latest master and reopen if still needed.

/close

**Comment by alculquicondor** [comment]
Oops, I forgot to click the "Submit review" button. Consider that if you open a new PR.

**Pull request closed without merging**